src/index.js

```diff
@@ -0,0 +1,177 @@
+
+/**********************1*********************/
+
+
+
+
+const data = [
+  { value: "1", label: "One" },
+  { value: "2", label: "Two" },
+  { value: "3", label: "Three" },
+];
+
+const values = data.reduce((values, { value }) => {
+  values.push(value);
+  return values;
+}, []);
+
+
+
+/**********************2*********************/
+
+
+async function getIndexes() {
+  return await fetch('https://api.coingecko.com/api/v3/indexes').then(res => res.json());
+}
+
+async function analyzeIndexes() {
+  const indexes = await getIndexes().catch(_ => {
+    throw new Error('Unable to fetch indexes');
+  });
+  return indexes;
+}
+
+
+
+/**********************3*********************/
+
+
+
+let state;
+const user = getUser();
+if (user) {
+  const project = getProject(user.id);
+  state = {
+    user,
+    project
+  };
+} else {
+  state = {
+    user: null,
+    project: null
+  };
+}
+ctx.body = state;
+
+
+
+/**********************4*********************/
+
+
+
+function getQueryProvider() {
+  const url = window.location.href;
+  const [_, provider] = url.match(/provider=([^&]*)/);
+  if (provider) {
+    return provider;
+  }
+  return;
+}
+
+
+
+/*********************5**********************/
+
+
+
+
+function getParagraphTexts() {
+  const texts = [];
```

Comment:
you're not altering the p element before pushing it to the texts array, so might as well return the array as it is

should be :
return document.querySelectorAll("p");